src/Video/PostProcessing/FXAAFilter.cpp

```diff
@@ -28,9 +28,13 @@ ShaderProgram* FXAAFilter::GetShaderProgram() const {
 
 void FXAAFilter::SetUniforms() {
     glUniform2fv(screenSizeLocation, 1, &screenSize[0]);
-    glUniform1f(brightnessLocation, brightness);
+    glUniform1f(gammaLocation, gamma);
 }
 
 void FXAAFilter::SetScreenSize(const glm::vec2& screenSize) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The function 'SetScreenSize' is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)